oci/memory_store.go

```diff
@@ -24,18 +24,26 @@ func (c *memoryStore) Add(id string, cont *Container) {
 }
 
 // Get returns a container from the store by id.
-func (c *memoryStore) Get(id string) *Container {
+func (c *memoryStore) Get(idOrName string) *Container {
```

Comment:
Okay, I reverted this and moved the server's getContainerFromRequest() function to ContainerServer